### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by tinosulzer**
Add basic skeleton for documentation

### CODE REVIEW

**Comment by tinosulzer** [comment]
Clicking on "Examples" in the sidebar
Screen Shot 2020-11-27 at 12 13 51 AM
takes me to this page
Screen Shot 2020-11-27 at 12 13 37 AM
which I don't know how to fix.

Also, don't know how to fix the failing test

File: Project.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ version = "4.15.0"
 BandedMatrices = "aae01518-5342-5314-be14-df237901396f"
 BlockBandedMatrices = "ffab5731-97b5-5995-9138-79e8c1846df0"
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```
Comment (ChrisRackauckas) [inline Project.toml:10]: Documenter shouldn't be a requirement. This should be only in the separate doc Project toml

File: Project.toml [hunk 2]
```diff
@@ -21,6 +22,7 @@ SuiteSparse = "4607b0f0-06f3-5cda-b6b1-a6196a1729e9"
 BandedMatrices = "0.15.11"
 BlockBandedMatrices = "0.8.6, 0.9"
 DiffEqBase = "6.4.1"
+Documenter = "0.25.3"
```
Comment (ChrisRackauckas) [inline Project.toml:25]:
```suggestion
```

File: Project.toml (hunk 1 above)
Comment (ChrisRackauckas) [inline Project.toml:10]:
```suggestion
```

File: docs/Project.toml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[deps]
+DiffEqOperators = "9fdde737-9c7f-55bf-ade8-46b3f136cc48"
```
Comment (ChrisRackauckas) [inline docs/Project.toml:2]:
```suggestion
[deps]
DiffEqOperators = "9fdde737-9c7f-55bf-ade8-46b3f136cc48"
Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

File: .github/workflows/Documentation.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1.4'
```
Comment (ChrisRackauckas) [inline .github/workflows/Documentation.yml:17]:
```suggestion
          version: '1'
```

**Pull request merged**